subsys/mgmt/mcumgr/lib/util/src/zcbor_bulk.c

```diff
@@ -13,24 +13,23 @@ int zcbor_map_decode_bulk(zcbor_state_t *zsd, struct zcbor_map_decode_key_val *m
 	size_t map_size, size_t *matched)
 {
 	bool ok;
```

Comment:
```suggestion
	bool ok = true;
```
Can be set here

Reply:
Usually ppl point out on my PRs that I initialize variables to early.

Reply:
I think it doesn't need to be initialized at all, since `ok = zcbor_tstr_decode(zsd, &key);` is always run before `ok` is checked.